packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -1991,6 +1993,7 @@ describe('network stubbing', function () {
             .intercept(`${url}*`, (req) => {
               // @ts-ignore
               req.on(eventName, (res) => {
+                res.headers['content-type'] = 'application/json'
```

Comment:
Is this a change all consumers will need to make that we need to document or were we incorrectly implementing this?

Reply:
@mschile Can you speak to why this change was necessary?

Reply:
No, consumers won't need to do this change. During the session lifecycle, we visit `about:blank` which sets the `__cypress.initial` cookie to ensure the correct injection level for subsequent requests. Without setting the `content-type`, it defaults to `text/html` which causes Cypress to believe this is the initial html request and do a `full` injection. By setting the `content-type` to `application/json`, we know not to inject into this response. Consumers won't need this change since they should be visiting between tests.